.github/workflows/release-publish.yml

```
@@ -231,13 +231,18 @@ jobs:
         mv nessie-${RELEASE_VERSION}.tgz nessie-helm-${RELEASE_VERSION}.tgz
         echo ::set-output name=NESSIE_HELM_CHART::nessie-helm-${RELEASE_VERSION}.tgz
 
+      # Publish Nessie Helm chart to Helm Repo
```

```suggestion
    # Publish Nessie Helm chart to Helm Repo
```
Indent should be fixed (for IDEs)

---

fixed